Gemfile.lock.release

```diff
@@ -559,6 +567,7 @@ GEM
       bigdecimal
       rexml
     crass (1.0.6)
+    csv (3.3.0)
```

Comment:
Are explicitly adding bigdecimal and csv now?  Expected?

Reply:
That's why I wanted the review - csv and bigdecimal are getting pulled in from other gems, so it is expected..I'm just not sure those specific versions will match the shipping Ruby or do we have to play games like we did previously?

Reply:
This is the games I'm referring to: https://github.com/ManageIQ/manageiq/blob/radjabov/Gemfile#L87-L93

I'm thinking we can just merge this and then see what happens on the build side? 🤷

Reply:
httparty is now bringing in csv below so I think this is ok.

Reply:
yeah, there's no easy way to figure out if there's incompatibilities going from default gems bigdecimal/csv to versioned ones on rubygems.org.  Moving off of default gems to indirect dependencies should at least allow us to detect security issues in those gems from bundler audit.  

I think our usage of csv and bigdecimal is much less than time, uri, and cgi so I agree, let's let it build and see what shakes out.